src/main/java/pl/techlab24/OSKManager/model/validation/ClientValidator.java

```diff
@@ -116,27 +119,27 @@ private static String validateDocumentNumber(String documentNumber) {
 
     private static boolean checkSum(String pesel) {
         int lastDigit;
-        int[] peselNumbers =  new int[11];
+        int[] peselNumbers = new int[11];
 
         for (int i = 0; i < pesel.length(); i++) {
             peselNumbers[i] = Integer.parseInt(String.valueOf(pesel.charAt(i)));
```

Comment:
You can convert for loop into: 
int[] peselNumbers = pesel.chars()
.map(x -> x - '0')
.toArray();